cmd/docker/main.go

```diff
@@ -176,7 +176,28 @@ func startServerWithOptions(cli *CliOptions) *apiserver.Server {
 		Version: "1.22", //dockerversion.Version,
 	}
 
-	tlsConfig := tlsconfig.ServerDefault
+	copyTLSConfig := func(c *tls.Config) *tls.Config {
```

Comment:
Could be, will do that. I added this as a temporary workaround until the library we use fixes their code :)